### PR TITLE
Orbitsoft Bid Adapter : add adapter back to current version

### PR DESCRIPTION
**Pull request by VitalyOrbit**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

File: modules/orbitsoftBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      // Sizes params (not supports by server, for future features)
+      let sizesParams = bidRequest.sizes;
+      let parsedSizes = utils.parseSizesInput(sizesParams);
+
+      serverRequests.push({
+        method: 'GET',
+        url: requestUrl,
+        data: Object.assign({
+          'scid': placementId,
+          'callback_uid': callbackId,
```
[the remark this answers is not in the document]
Comment (patmmccann) [reply modules/orbitsoftBidAdapter.js:96]: please note this alert

[the remark this answers is not in the document]
Comment (VitalyOrbit) [reply modules/orbitsoftBidAdapter.js]: Could you clarify this alert? The `callbackId` gets its value from the `utils.getUniqueIdentifierStr()` method. This method uses in other modules too.

File: modules/orbitsoftBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+          customParamsArray['c.' + customField] = customParams[customField];
+        }
+      }
+
+      // Sizes params (not supports by server, for future features)
+      let sizesParams = bidRequest.sizes;
+      let parsedSizes = parseSizesInput(sizesParams);
+      let requestData = Object.assign({
+        'scid': placementId,
+        'callback_uid': getUniqueIdentifierStr(),
```
[the remark this answers is not in the document]
Comment (VitalyOrbit) [reply modules/orbitsoftBidAdapter.js:91]: What changes I have to do? Could anyone answer me? The `getUniqueIdentifierStr()` function from *utils* is used. This function is also used in other modules. Why can't I use it here?

[the remark this answers is not in the document]
Comment (VitalyOrbit) [reply modules/orbitsoftBidAdapter.js]: @ChrisHuie I don't use that function any more. And also I added the "adomain". Are there any other issues or it is OK now?

**Comment by VitalyOrbit** [comment]
@ChrisHuie @dangoldin  could you please review this PR?

**Comment by ChrisHuie** [comment]
> @ChrisHuie @dangoldin could you please review this PR?

Yes. Reviewing this week. 👍

**Review by ChrisHuie — APPROVED** [review]

**Comment by ChrisHuie** [comment]
@VitalyOrbit approved and merging before this weeks release. Can you please add back your docs to our docs repo please 🙏 Believe we removed the docs for this as well.

**Comment by VitalyOrbit** [comment]
@ChrisHuie could you clarify what the docs should be updated? And where is your docs repo?
1. https://github.com/prebid/docs
2. https://github.com/prebid/prebid.github.io (this repo already contains https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/orbitsoft.md)
3. Another place

**Comment by ChrisHuie** [comment]
> @ChrisHuie could you please clarify what docs should be updated? And where is your docs repo?
> 
> 1. https://github.com/prebid/docs
> 2. https://github.com/prebid/prebid.github.io (this repo already contains https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/orbitsoft.md)
> 3. Another place

If its in #2 we are good 👍

**Comment by VitalyOrbit** [comment]
@ChrisHuie have you already released?
I gt an error on downloading with the Orbisoft adapter. I selected only this one.

`{"error":"Prebid file not built properly","requestId":"8d100b35-80f5-47da-a2ce-5324e893d98f"}`

**Pull request merged**